connection/src/test/java/com/netcracker/ConnectionTest.java

```diff
@@ -0,0 +1,42 @@
+package com.netcracker;
+
+import com.netcracker.domain.Message;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.SocketAddress;
+
+import static com.netcracker.domain.MessageType.TEXT;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class ConnectionTest {
+    private final Connection testConnection = mock(Connection.class);
```

Comment:
&#x1F34E; **Issue**
You should test a real object instead of a mock. It doesn't make sense to test a mock. Mock is an object without its own behavior.